modules/installation-guide/partials/proc_using-codeready-containers-to-set-up-openshift-4.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// installing-{prod-id-short}-on-codeready-containers
+
+[id="using-codeready-containers-to-set-up-openshift-4_{context}"]
+= Using CodeReady Containers to configure OpenShift 4
+
+This section describes how to use CodeReady Containers to configure OpenShift 4.
+
+.Prerequisites
+
+* An installation of CodeReady Containers. See link:https://cloud.redhat.com/openshift/install/crc/installer-provisioned[Installing CodeReady Containers].
+
+.Procedure
+
+. Configure your host machine for CodeReady Containers:
++
+----
+$ crc setup
+----
+
+. Remove any previous cluster:
++
+----
+$ crc delete
+----
+
+. Start the CodeReady Containers virtual machine with at least 12 GB of RAM:
++
+----
+$ crc start --memory 12288
+----
+
+. When prompted, supply your user pull secret.
+. Take note of the password for the user `kubedmin` that is displayed at the end of the installation.
+. Access the OpenShift web console:
++
+----
+$ crc console
+----
+
+. Log in for the first time with the `developer` account (password: `developer`) to initialize the first user using OAuth:
++
+----
+$ oc login --username="developer" --password="developer"
+----
+
+. Log out:
++
+----
+$ oc logout
+----
+
+. Log in again with the previously mentioned `kubedmin` user and password:
++
+----
+$ oc login --username="kubeadmin" --password="__<kubeadmin-password>__"
+----
+
+. Follow the procedure for xref:installing-che-on-openshift-4-using-operatorhub.adoc[].
+
+.Additional resources
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_codeready_containers/[Product documentation for Red Hat CodeReady Containers].
+* link:https://github.com/code-ready/crc[CodeReady Containers on GitHub].
```

Comment:
This is a very well written procedure :-)

Reply:
Ano ano, dělal jsem to já :D